lms/services/token_store.py

```diff
@@ -61,3 +59,9 @@ def get(self):
                 explanation="We don't have a Canvas API access token for this user",
                 response=None,
             ) from err
+
+
+def token_store_service_factory(_context, request):
+    return TokenStore(
+        request.db, request.lti_user.oauth_consumer_key, request.lti_user.user_id
+    )
```

Comment:
It's nice that the code for creating a `TokenStore` and injecting its dependencies lives alongside `TokenStore` itself in `token_store.py`